### PR TITLE
New Post: 2021 Scroll Survey Report

### LINKED ISSUE #5622: [A11y][CSS] 2021 Scroll Survey Results
* [Draft](https://docs.google.com/document/d/1c6Ub_4gzFOEa-UZqYVFYIi1IlQZvg1-aR-WUmKmaJds/edit?resourcekey=0-SykHouVTEBiPgw2RbbfjIA#)

### PR DESCRIPTION
**Pull request by argyleink**
Fixes #5622

To do:
- [x] use the web.dev cdn to host the report link
- [x] get final reviews
- [x] update date

### CODE REVIEW

File: src/site/content/en/blog/2021-scroll-survey-report/index.md [hunk 1]
```diff
@@ -0,0 +1,397 @@
...
+      from the survey
+    </p>
+  </div>
+</div>
+
+Some of the survey questions asked about usage of certain APIs with a possible
+answer of Yes, No, or "today I learned". One notable piece of feedback was the
+number of people who reported learning about `touch-action` from the survey, as
+it's a critical property when building custom touch gestures that need to
+interact within scroll.
```
Comment (rachelandrew) [inline src/site/content/en/blog/2021-scroll-survey-report/index.md:153]:
```suggestion
Some of the survey questions asked about using certain APIs with a possible
answer of Yes, No, or "today I learned." One notable piece of feedback was the
number of people who reported learning about `touch-action` from the survey, as
it's a critical property when building custom touch gestures that need to
interact within scroll.
```

File: src/site/content/en/blog/2021-scroll-survey-report/index.md [hunk 2]
```diff
@@ -0,0 +1,397 @@
...
+  {% Video
+    src="video/vS06HQ1YTsbMKSFTIPl2iogUQP73/xLwqQ6xGBIdI0uxFx566.mp4",
+    autoplay="true",
+    loop="true",
+    width="380",
+    muted="true"
+  %}
+  <figcaption class="w-figure">
+    Video shows cyclical seconds scrolling, <br>
+    where after 60 it's begins at 0 again
```
Comment (rachelandrew) [inline src/site/content/en/blog/2021-scroll-survey-report/index.md:179]:
```suggestion
    The video shows cyclical seconds scrolling, <br>
    after 60 seconds it begins at 0 again.
```

File: src/site/content/en/blog/2021-scroll-survey-report/index.md [hunk 3]
```diff
@@ -0,0 +1,397 @@
...
+    <p class="w-stat__desc">
+      report <b>not at all</b> <br>
+      or <b>slightly important</b>
+    </p>
+  </div>
+</div>
+
+Respondents felt strong about the importance of scrollable areas. Another signal
+of the sentiment behind the struggles required to deliver high quality
+scrollable areas.
```
Comment (rachelandrew) [inline src/site/content/en/blog/2021-scroll-survey-report/index.md:212]:
```suggestion
Respondents felt strongly about the importance of scrollable areas, 
giving another signal about the struggles required to deliver high-quality scrolling.
```

File: src/site/content/en/blog/2021-scroll-survey-report/index.md [hunk 4]
```diff
@@ -0,0 +1,397 @@
...
+    <p class="w-stat__desc">
+      <b>somewhat</b> or <br>
+      <b>extremely difficult</b>
+    </p>
+  </div>
+</div>
+
+Two thirds of respondents deliver infinite scroll in their web work and an equal
+amount report it's difficult to do. Another example of high usage paired with
+high difficulty, which indicates to us an area needing attention.
```
Comment (rachelandrew) [inline src/site/content/en/blog/2021-scroll-survey-report/index.md:261]:
```suggestion
Two-thirds of respondents deliver infinite scroll in their web work, and an equal
amount report it's difficult to do. Another example of high usage paired with
high difficulty, which indicates to us an area needing attention.
```

File: src/site/content/en/blog/2021-scroll-survey-report/index.md [hunk 5]
```diff
@@ -0,0 +1,397 @@
...
+    <p class="w-stat__desc">
+      report <b>somewhat</b> or <br>
+      <b>extremely difficult</b>
+    </p>
+  </div>
+</div>
+
+Almost half of all respondents use scroll-orchestrated animations and half the
+respondents find it difficult. Another example of high usage paired with high
+difficulty.
```
Comment (rachelandrew) [inline src/site/content/en/blog/2021-scroll-survey-report/index.md:291]:
```suggestion
Almost half of all respondents use scroll-orchestrated animations and half the
respondents find it difficult, once again linking high usage with difficulty.
```

File: src/site/content/en/blog/2021-scroll-survey-report/index.md [hunk 6]
```diff
@@ -0,0 +1,397 @@
...
+    <p class="w-stat__figure">50<sub class="w-stat__sub">%</sub></p>
+    <p class="w-stat__desc">
+      <b>sometimes</b>
+    </p>
+  </div>
+</div>
+
+The built-in scroll and touch interactions of phone and tablet applications are
+often touted as a clear place where the web can catch up. The features include
+scroll linked animations, programmatic interfaces, voice integration, scroll
```
Comment (rachelandrew) [inline src/site/content/en/blog/2021-scroll-survey-report/index.md:315]:
```suggestion
scroll-linked animations, programmatic interfaces, voice integration, scroll
```

File: src/site/content/en/blog/2021-scroll-survey-report/index.md [hunk 7]
```diff
@@ -0,0 +1,397 @@
...
+The survey results are segmented into four categories: 
+[compatibility](#compatibility), 
+[education](#education),
+[APIs](#apis), 
+and [features](#features).
+
+### Compatibility
+
+The Chrome team has [declared a goal](https://web.dev/compat2021) to decrease
+the number of web compatibility issues, including scroll compatibility
```
Comment (rachelandrew) [inline src/site/content/en/blog/2021-scroll-survey-report/index.md:341]:
```suggestion
The Chrome team has [declared a goal](https://web.dev/compat2021) to decrease
the number of web compatibility issues, including scroll compatibility.
```

File: src/site/content/en/blog/2021-scroll-survey-report/index.md [hunk 8]
```diff
@@ -0,0 +1,397 @@
...
+1. `overscroll-behavior` cross browser.
+1. Removing prefixes from `-webkit-scrollbar` and following the standard.
+
+### Education
+
+The survey results showed that there needs to be more education around
+`touch-action` and [logical
+properties](https://web.dev/logical-property-shorthands/). The browser is at the
+forefront of international layout and it's apparent it's under utilized or
+misunderstood.
```
Comment (rachelandrew) [inline src/site/content/en/blog/2021-scroll-survey-report/index.md:354]:
```suggestion
forefront of international layout, and it's apparent it's underutilized or
misunderstood.
```

File: src/site/content/en/blog/2021-scroll-survey-report/index.md [hunk 9]
```diff
@@ -0,0 +1,397 @@
...
+Areas to focus on:
+1. [`touch-action`](https://developer.mozilla.org/en-US/docs/Web/CSS/touch-action)
+1. [Logical properties](https://web.dev/learn/css/logical-properties/)
+
+### APIs
+
+Usage of scroll snapping is growing, and developers have responded that they
+want the ability to use features interoperably with popular libraries and
+plugins. Shrinking this gap between CSS and plugin libraries will help the
+satisfaction of scroll snap developer and user experience.
```
Comment (rachelandrew) [inline src/site/content/en/blog/2021-scroll-survey-report/index.md:365]:
```suggestion
Usage of scroll snapping is growing, and developers have responded that they
want to use features interoperably with popular libraries and
plugins. Shrinking this gap between CSS and plugin libraries will help the
satisfaction of scroll snap developer and user experience.
```

File: src/site/content/en/blog/2021-scroll-survey-report/index.md [hunk 10]
```diff
@@ -0,0 +1,397 @@
...
+
+We'll focus API work on `scroll-snap`:
+1. API availability / compatibility across browsers.
```
Comment (rachelandrew) [inline src/site/content/en/blog/2021-scroll-survey-report/index.md:368]:
```suggestion
1. API availability and compatibility across browsers.
```

File: src/site/content/en/blog/2021-scroll-survey-report/index.md [hunk 11]
```diff
@@ -0,0 +1,397 @@
...
+experience of built-in scrolling. 
+
+### Overall satisfaction building scroll interactions on the web
+
+<small>Question 27</small>
+
+{% Img src="image/vS06HQ1YTsbMKSFTIPl2iogUQP73/ycHj33nZb1KfUFxNBdH5.png", alt="A
+pie chart showing 5 sections: 6.3% extremely dissatisfied, 2.7% extremely
+satisfied, 23.4% somewhat satisfied, 28.8% neither satisfied nor dissatisfied,
+38.7% somewhat dissatisfied", width="800", height="400" %}
```
Comment (rachelandrew) [inline src/site/content/en/blog/2021-scroll-survey-report/index.md:328]:
```suggestion
38.7% somewhat dissatisfied.", width="800", height="400" %}
```

File: src/site/content/en/blog/2021-scroll-survey-report/index.md [hunk 12]
```diff
@@ -0,0 +1,397 @@
...
+    <p class="w-stat__desc">
+      use it <b>sometimes</b> <br>
+      to <b>every project</b>
+    </p>
+  </div>
+  <div class="w-stat">
+    <p class="w-stat__figure">60<sub class="w-stat__sub">%</sub></p>
+    <p class="w-stat__desc">
+      <b>somewhat</b> or <br>
+      <b>extremely difficult</b>
```
Comment (rachelandrew) [inline src/site/content/en/blog/2021-scroll-survey-report/index.md:254]:
```suggestion
      <b>extremely difficult</b>.
```

File: src/site/content/en/blog/2021-scroll-survey-report/index.md [hunk 13]
```diff
@@ -0,0 +1,397 @@
...
+### Overall satisfaction with web scrolling
+
+<small>Question 27</small>
+
+<div class="w-stats">
+  <div class="w-stat">
+    <p class="w-stat__figure">45<sub class="w-stat__sub">%</sub></p>
+    <p class="w-stat__desc">
+      are <b>somewhat</b> or <b>extremely dissatisfied overall</b> <br>
+      with <b>web scrolling</b>
```
Comment (rachelandrew) [inline src/site/content/en/blog/2021-scroll-survey-report/index.md:51]:
```suggestion
      with <b>web scrolling</b>.
```

File: src/site/content/en/blog/2021-scroll-survey-report/index.md [hunk 14]
```diff
@@ -0,0 +1,397 @@
...
+
+<small>Question 2</small>
+
+<div class="w-stats">
+  <div class="w-stat">
+    <p class="w-stat__figure">43<sub class="w-stat__sub">%</sub></p>
+    <p class="w-stat__desc">
+      reported it's <b>somewhat</b> or <br>
+      <b>extremely difficult <br>
+      to  work with scrolling</b>
```
Comment (rachelandrew) [inline src/site/content/en/blog/2021-scroll-survey-report/index.md:74]:
```suggestion
      to  work with scrolling</b>.
```

File: src/site/content/en/blog/2021-scroll-survey-report/index.md [hunk 15]
```diff
@@ -0,0 +1,397 @@
...
+
+<small>Question 3</small>
+
+<div class="w-stats">
+  <div class="w-stat">
+    <p class="w-stat__figure">51<sub class="w-stat__sub">%</sub></p>
+    <p class="w-stat__desc">
+      report <b>touch interactions</b> as <br>
+      <b>very</b> or <b>extremely important</b> <br>
+      to their work
```
Comment (rachelandrew) [inline src/site/content/en/blog/2021-scroll-survey-report/index.md:103]:
```suggestion
      to their work.
```

File: src/site/content/en/blog/2021-scroll-survey-report/index.md [hunk 16]
```diff
@@ -0,0 +1,397 @@
...
+### Difficulty of tab key or gamepad navigation
+
+<small>Question 5a</small>
+
+<div class="w-stats">
+  <div class="w-stat">
+    <p class="w-stat__figure">44<sub class="w-stat__sub">%</sub></p>
+    <p class="w-stat__desc">
+      report <b>somewhat</b> or <b>extremely difficult</b> <br>
+      to do <b>gamepad</b> and <b>tab navigation</b>
```
Comment (rachelandrew) [inline src/site/content/en/blog/2021-scroll-survey-report/index.md:124]:
```suggestion
      to do <b>gamepad</b> and <b>tab navigation</b>.
```

File: src/site/content/en/blog/2021-scroll-survey-report/index.md [hunk 17]
```diff
@@ -0,0 +1,397 @@
...
+
+<small>Question 9</small>
+
+<div class="w-stats">
+  <div class="w-stat">
+    <p class="w-stat__figure">50<sub class="w-stat__sub">%</sub></p>
+    <p class="w-stat__desc">
+      report <b>learning</b> about <br>
+      <b>`touch-action: manipulation`</b> <br>
+      from the survey
```
Comment (rachelandrew) [inline src/site/content/en/blog/2021-scroll-survey-report/index.md:144]:
```suggestion
      from the survey.
```

File: src/site/content/en/blog/2021-scroll-survey-report/index.md [hunk 18]
```diff
@@ -0,0 +1,397 @@
...
+### Cyclical scrolling
+
+<small>Question 27</small>
+
+<div class="w-stats">
+  <div class="w-stat">
+    <p class="w-stat__figure">58<sub class="w-stat__sub">%</sub></p>
+    <p class="w-stat__desc">
+      report <b>sometimes, often</b> or on <b>every project</b> <br>
+      using <b>cyclical scrolling</b>
```
Comment (rachelandrew) [inline src/site/content/en/blog/2021-scroll-survey-report/index.md:164]:
```suggestion
      using <b>cyclical scrolling</b>.
```

File: src/site/content/en/blog/2021-scroll-survey-report/index.md [hunk 19]
```diff
@@ -0,0 +1,397 @@
...
+
+### Carousels
+
+<small>Question 20</small>
+
+<div class="w-stats">
+  <div class="w-stat">
+    <p class="w-stat__figure">87<sub class="w-stat__sub">%</sub></p>
+    <p class="w-stat__desc">
+      <b>have used</b> carousels
```
Comment (rachelandrew) [inline src/site/content/en/blog/2021-scroll-survey-report/index.md:222]:
```suggestion
      <b>have used</b> carousels.
```

File: src/site/content/en/blog/2021-scroll-survey-report/index.md [hunk 20]
```diff
@@ -0,0 +1,397 @@
...
+    </p>
+  </div>
+  <div class="w-stat">
+    <p class="w-stat__figure">24<sub class="w-stat__sub">%</sub></p>
+    <p class="w-stat__desc">
+      report they're <br>
+      <b>easy</b> to manage
```
Comment (rachelandrew) [inline src/site/content/en/blog/2021-scroll-survey-report/index.md:229]:
```suggestion
      <b>easy</b> to manage.
```

**Review by rachelandrew — CHANGES_REQUESTED** [review]
I've done some copy editing on this, the main thing is that it is currently missing an image.

File: src/site/content/en/blog/2021-scroll-survey-report/index.md [hunk 21]
```diff
@@ -0,0 +1,397 @@
+---
+layout: post
+title: 2021 Scroll Survey Report
+subhead: Get the 2021 Scroll Survey Report plus words from the Chrome team about how this impacts priorities and plans for Chromium and the web.
+authors:
+  - adamargyle
+  - samdutton
+description: Get the 2021 Scroll Survey Report plus words from the Chrome team about how this impacts priorities and plans for Chromium and the web.
```
[the remark this answers is not in the document]
Comment (samdutton) [reply src/site/content/en/blog/2021-scroll-survey-report/index.md:8]: I'm OK with not having one for this, but if we definitely need an image, we could get something from unsplash.com. I like the way it stands out (at the moment) without a thumbnail/hero.

**Comment by samdutton** [comment]
Good edits @rachelandrew  — thank you!

File: src/site/content/en/blog/2021-scroll-survey-report/index.md (hunk 21 above)
[the remark this answers is not in the document]
Comment (argyleink) [reply src/site/content/en/blog/2021-scroll-survey-report/index.md:8]: yeah, i intentionally left an image out, thought it worked good. but am willing to find one if required 👍🏻

[the remark this answers is not in the document]
Comment (rachelandrew) [reply src/site/content/en/blog/2021-scroll-survey-report/index.md:8]: I believe we can have a thumbnail without a hero, which would give it a sharing image without having the big image at the top of the post.

File: src/site/content/en/blog/2021-scroll-survey-report/index.md [hunk 22]
```diff
@@ -0,0 +1,394 @@
...
+  - blog
+  - css
+  - dom
+  - javascript
+  - mobile
+  - ux
+---
+
+In April, the Chrome team [released a scroll and touch-action
+survey](https://web.dev/2021-scroll-survey/) based on top reported issues from
```
Comment (rachelandrew) [inline src/site/content/en/blog/2021-scroll-survey-report/index.md:20]:
```suggestion
survey](/2021-scroll-survey/) based on top reported issues from
```

File: src/site/content/en/blog/2021-scroll-survey-report/index.md [hunk 23]
```diff
@@ -0,0 +1,394 @@
...
+      <b>extremely difficult</b>.
+    </p>
+  </div>
+</div>
+
+Two-thirds of respondents deliver infinite scroll in their web work, and an equal
+amount report it's difficult to do. Another example of high usage paired with
+high difficulty, which indicates to us an area needing attention.
+
+While [`content-visibility`](https://web.dev/content-visibility/) and
```
Comment (rachelandrew) [inline src/site/content/en/blog/2021-scroll-survey-report/index.md:262]:
```suggestion
While [`content-visibility`](/content-visibility/) and
```

File: src/site/content/en/blog/2021-scroll-survey-report/index.md [hunk 24]
```diff
@@ -0,0 +1,394 @@
...
+
+The survey results showed that there needs to be more education around
+`touch-action` and [logical
+properties](https://web.dev/logical-property-shorthands/). The browser is at the
+forefront of international layout, and it's apparent it's underutilized or
+misunderstood.
+
+Areas to focus on:
+1. [`touch-action`](https://developer.mozilla.org/en-US/docs/Web/CSS/touch-action)
+1. [Logical properties](https://web.dev/learn/css/logical-properties/)
```
Comment (rachelandrew) [inline src/site/content/en/blog/2021-scroll-survey-report/index.md:356]:
```suggestion
The Chrome team has [declared a goal](/compat2021) to decrease
the number of web compatibility issues, including scroll compatibility.

The first three compatibility issues to focus on:
1. Horizontal scrolling compatibility.
1. `overscroll-behavior` cross browser.
1. Removing prefixes from `-webkit-scrollbar` and following the standard.

### Education

The survey results showed that there needs to be more education around
`touch-action` and [logical
properties](/logical-property-shorthands/). The browser is at the
forefront of international layout, and it's apparent it's underutilized or
misunderstood.

Areas to focus on:
1. [`touch-action`](https://developer.mozilla.org/en-US/docs/Web/CSS/touch-action)
1. [Logical properties](/learn/css/logical-properties/)
```

File: src/site/content/en/blog/2021-scroll-survey-report/index.md [hunk 25]
```diff
@@ -0,0 +1,394 @@
...
+1. Infinite scroll
+
+### Resources
+
+- [Scroll Survey
+  Report](https://drive.google.com/file/d/10WXTJHCZmH0rXEh3J7z3ki96-f9SHs_R/view?usp=sharing&resourcekey=0-5b1y_6rZDhyt6Tkbz3FCzA)
+- [Survey Announcement](https://web.dev/2021-scroll-survey/)
+- [Mozilla DNA Reports](https://insights.developer.mozilla.org/)
+- [Compat2021: Eliminating five top compatibility pain points on the
+  web](https://web.dev/compat2021/)
```
Comment (rachelandrew) [inline src/site/content/en/blog/2021-scroll-survey-report/index.md:392]:
```suggestion
- [Scroll Survey
  Report](https://drive.google.com/file/d/10WXTJHCZmH0rXEh3J7z3ki96-f9SHs_R/view?usp=sharing&resourcekey=0-5b1y_6rZDhyt6Tkbz3FCzA)
- [Survey Announcement](/2021-scroll-survey/)
- [Mozilla DNA Reports](https://insights.developer.mozilla.org/)
- [Compat2021: Eliminating five top compatibility pain points on the
  web](/compat2021/)
```

File: src/site/content/en/blog/2021-scroll-survey-report/index.md (hunk 21 above)
[the remark this answers is not in the document]
Comment (samdutton) [reply src/site/content/en/blog/2021-scroll-survey-report/index.md:8]: Maybe a silly scroll pun: https://unsplash.com/photos/aXeVH4FcS1k

...or something more generic: https://unsplash.com/photos/Hcfwew744z4

[the remark this answers is not in the document]
Comment (samdutton) [reply src/site/content/en/blog/2021-scroll-survey-report/index.md:8]: @argyleink @rachelandrew 

Any preferences for a thumbnail?

[the remark this answers is not in the document]
Comment (argyleink) [reply src/site/content/en/blog/2021-scroll-survey-report/index.md:8]: i'm down with the pun 👍🏻

[the remark this answers is not in the document]
Comment (samdutton) [reply src/site/content/en/blog/2021-scroll-survey-report/index.md:8]: 👍 I'll add it as thumbnail.

File: src/site/content/en/blog/2021-scroll-survey-report/index.md [hunk 26]
```diff
@@ -0,0 +1,398 @@
...
+
+From our research, these difficulties come from the multitude of use cases for
+scroll. When we talk about scrolling, that might include:
+- [Positioning elements within scrollable areas.](https://elad.medium.com/css-position-sticky-how-it-really-works-54cd01dc2d46)
+- [Infinite scroll](https://blog.logrocket.com/infinite-scroll-techniques-in-react-adcfd7ff32bd/)
+- [Scroll linked animation.](https://greensock.com/scrolltrigger/)
+- [Carousels](https://css-tricks.com/css-only-carousel/)
+- [Scrollview padding](https://blog.alexandergottlieb.com/overflow-scroll-and-the-right-padding-problem-a-css-only-solution-6d442915b3f4)
+- [Cyclical scroll](https://www.magictoolbox.com/magicscroll/integration/)
+- [Virtualized scroll.](https://github.com/tbranyen/hyperlist)
```
Comment (rachelandrew) [inline src/site/content/en/blog/2021-scroll-survey-report/index.md:89]:
```suggestion
- [Positioning elements within scrollable areas](https://elad.medium.com/css-position-sticky-how-it-really-works-54cd01dc2d46)
- [Infinite scroll](https://blog.logrocket.com/infinite-scroll-techniques-in-react-adcfd7ff32bd/)
- [Scroll linked animation](https://greensock.com/scrolltrigger/)
- [Carousels](https://css-tricks.com/css-only-carousel/)
- [Scrollview padding](https://blog.alexandergottlieb.com/overflow-scroll-and-the-right-padding-problem-a-css-only-solution-6d442915b3f4)
- [Cyclical scroll](https://www.magictoolbox.com/magicscroll/integration/)
- [Virtualized scroll](https://github.com/tbranyen/hyperlist)
```

File: src/site/content/en/blog/2021-scroll-survey-report/index.md [hunk 27]
```diff
@@ -0,0 +1,398 @@
...
+**View the [2021 Scroll Survey
+Report](https://storage.googleapis.com/web-dev-uploads/file/vS06HQ1YTsbMKSFTIPl2iogUQP73/QZopyELSk8T7IpsgOnRU.pdf).**
+
+## Noteworthy results
+
+The survey anonymously collected 880 submissions, with 366 answering every
+question. 
+
+While getting started with scrolling is one line of CSS, like `overflow-x:
+scroll;`, the surface area of scroll APIs and options is large, spanning JS to
```
Comment (rachelandrew) [inline src/site/content/en/blog/2021-scroll-survey-report/index.md:40]:
```suggestion
scroll;`, the surface area of scroll APIs and options is large, spanning JavaScript to
```

File: src/site/content/en/blog/2021-scroll-survey-report/index.md [hunk 28]
```diff
@@ -0,0 +1,398 @@
...
+layout: post
+title: 2021 Scroll Survey Report
+subhead: Get the 2021 Scroll Survey Report plus words from the Chrome team about how this impacts priorities and plans for Chromium and the web.
+authors:
+  - adamargyle
+  - samdutton
+description: Get the 2021 Scroll Survey Report plus words from the Chrome team about how this impacts priorities and plans for Chromium and the web.
+date: 2021-08-02
+thumbnail: image/80mq7dk16vVEg8BBhsVe42n6zn82/0yI4zjveGNiC1Kh8QzhV.jpg
+alt: The unveiling of the new Torah display at Trinity International University, photo by Taylor Wilcox
```
Comment (rachelandrew) [inline src/site/content/en/blog/2021-scroll-survey-report/index.md:11]:
```suggestion
alt: The unveiling of the new Torah display at Trinity International University, photo by Taylor Wilcox.
```

**Review by rachelandrew — APPROVED** [review]

**Pull request merged**